### PR TITLE
Bug fixing

### PR DESCRIPTION
**Pull request by lavfrim**
fix complex bugs

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -33,6 +33,10 @@ body {
     font-family: var(--font-main);
 }
 
+header {
+    z-index: 10;
```
Comment (lavfrim) [inline style.css:37]: Could be less than 10 but more than 0

**Comment by lavfrim** [comment]
image

**Comment by lavfrim** [comment]
image

**Comment by kcheky** [comment]
super

**Review by kcheky — APPROVED** [review]

**Pull request merged**